DQMServices/Core/interface/DQMStore.h

```diff
@@ -753,6 +753,8 @@ namespace dqm {
       void debugTrackME(const char* message, MonitorElement* me_local, MonitorElement* me_global) const;
       // accesor to keep MEsToSave_ private
       const auto& getMEsToSave() const { return MEsToSave_; }
+      // accesor to keep onlineMode_ private
+      const bool& getMode() const { return onlineMode_; }
```

Comment:
I would have rather called the method `getOnlineMode()` so that the following line can become more understandable:
https://github.com/cms-sw/cmssw/pull/41736/files#diff-f9ea48cd2e3bd19ffa700c26566b06832095be6446c78486bd8612fe369b8b18R105

But ok, let merge as such (at least for the master) and I'll let you @jfernan2 decide whether it is worth to rename it